packages/synapse-interface/components/Portfolio/components/PortfolioTokenAsset.tsx

```diff
@@ -92,7 +114,7 @@ export const PortfolioTokenAsset = ({
   )
 }
 
-const getParsedBalance = (
+export const getParsedBalance = (
```

Comment:
Move this to utils since its not used in this file and it should sit there instead

Reply:
Updated in https://github.com/synapsecns/sanguine/pull/2401/commits/4ae15d7816a1d00d847ae0e26ce4dc88232fd4dd